[hairpin] fix argument of nsenter

CaoShuFeng opened the pull request:
**Release note**:
<!--  Steps to write your release note:
1. Use the release-note-* labels to set the release note state (if you have access) 
2. Enter your extended release note in the below block; leaving it blank means using the PR title as the release note. If no release note is required, just write `NONE`. 
-->
```release-note
None
```

We should use:
	nsenter --net=netnsPath -- -F some_command
instend of:
	nsenter -n netnsPath -- -F some_command
Because "nsenter -n netnsPath" get an error output:
	# nsenter -n /proc/67197/ns/net ip addr
	nsenter: neither filename nor target pid supplied for ns/net

If we really want use -n, we need to use -n in such format:
	# sudo nsenter -n/proc/67197/ns/net ip addr

k8s-reviewable commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.kubernetes.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/38101)
<!-- Reviewable:end -->

CaoShuFeng committed:
[hairpin] fix argument of nsenter

We should use:
	nsenter --net=netnsPath -- -F some_command
instend of:
	nsenter -n netnsPath -- -F some_command
Because "nsenter -n netnsPath" get an error output:
	# nsenter -n /proc/67197/ns/net ip addr
	nsenter: neither filename nor target pid supplied for ns/net

If we really want use -n, we need to use -n in such format:
	# sudo nsenter -n/proc/67197/ns/net ip addr

bboreham commented:
Note this code path is only called when using Rkt; the Docker path calls `SetUpContainerPid`

CaoShuFeng commented:
@bboreham 
Yes, it is.
But I failed to deploy an environment  with RKT. Otherwise I will commit a bug report.

thockin commented:
@freehan

freehan commented:
Thanks for capturing this. LGTM

freehan commented:
Need to run `hack/update-bazel.sh`

timstclair commented:
/approve

timstclair commented:
@k8s-bot bazel test this

CaoShuFeng commented:
Can someone ask aws e2e to test this?

ixdy commented:
@k8s-bot aws e2e test this

CaoShuFeng commented:
@k8s-bot bubemark e2e test this

ixdy commented:
@k8s-bot kubemark e2e test this
